src/diff-snapshot.js

```diff
@@ -18,7 +18,7 @@ const path = require('path');
 const pixelmatch = require('pixelmatch');
 const ssim = require('ssim.js');
 const { PNG } = require('pngjs');
-const rimraf = require('rimraf');
+const { rimrafSync } = require('rimraf');
```

Comment:
It would be better to remove the usage of `rimraf` from this file. See https://github.com/es-tooling/module-replacements/blob/main/docs/modules/rimraf.md for tips on how to do that

Reply:
Thanks for the suggestion, updated the code.